### PR TITLE
Sip plugins - custom headers in accept request

### PR DESCRIPTION
**Pull request by ihusejnovic**
Hi, 
regarding the question from my good friend and colleague Nadin on google group (https://groups.google.com/forum/#!topic/meetecho-janus/kEu7zvi2dFc), here is the implementation of custom headers in the accept request.

I have already signed CLA. 

Thank you.

### CODE REVIEW

File: plugins/janus_sip.c [hunk 1]
```diff
@@ -1100,6 +1104,32 @@ static void janus_sip_random_string(int length, char *buffer) {
 	}
 }
 
+static void parse_custom_headers(json_t *root, char *custom_headers) {
```
Comment (lminiero) [inline plugins/janus_sip.c]: Can you prefix this function name with `janus_sip_`? It's custom in all the Janus code.

File: plugins/janus_sipre.c [hunk 2]
```diff
@@ -157,6 +157,8 @@ static struct janus_json_parameter register_parameters[] = {
 	{"username", JANUS_JSON_STRING, JANUS_JSON_PARAM_REQUIRED},
 	{"secret", JANUS_JSON_STRING, 0},
 	{"authuser", JANUS_JSON_STRING, 0},
+	{"display_name", JSON_STRING, 0},
+        {"user_agent", JSON_STRING, 0}
```
Comment (lminiero) [inline plugins/janus_sipre.c]: Indentation is broken here (should be tabs).

File: plugins/janus_sipre.c [hunk 3]
```diff
@@ -936,6 +939,32 @@ static void janus_sipre_random_string(int length, char *buffer) {
 	}
 }
 
+static void parse_custom_headers(json_t *root, char *custom_headers) {
```
Comment (lminiero) [inline plugins/janus_sipre.c]: Same here for the prefix, but with `janus_sipre_`.

File: plugins/janus_sip.c (hunk 1 above)
Comment (ihusejnovic) [reply plugins/janus_sip.c]: Done.

**Comment by lminiero** [comment]
Thanks for the quick fixes, merging then!

File: plugins/janus_sipre.c (hunk 2 above)
Comment (ihusejnovic) [reply plugins/janus_sipre.c]: There was a missing comma, too. Fixed.

File: plugins/janus_sipre.c (hunk 3 above)
Comment (ihusejnovic) [reply plugins/janus_sipre.c]: Done.

**Pull request merged**